src/blocks/recommendations/components/parsely-recommendations-inspector-controls.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { InspectorControls } from '@wordpress/block-editor';
+import {
+	PanelBody,
+	PanelRow,
+	RadioControl,
+	RangeControl,
+	SelectControl,
+	TextControl,
+	ToggleControl,
+} from '@wordpress/components';
+
+const ParselyRecommendationsInspectorControls = ( {
+	attributes: { boost, imagestyle, limit, showimages, sort, tag, title },
+	setAttributes,
+} ) => {
+	return (
+		<InspectorControls>
+			<PanelBody title="Settings" initialOpen={ true }>
+				<PanelRow>
+					<TextControl
+						label={ __( 'Title', 'wp-parsely' ) }
+						value={ title }
+						onChange={ ( newval ) => setAttributes( { title: newval } ) }
+					/>
+				</PanelRow>
+				<PanelRow>
+					<RangeControl
+						label={ __( 'Maximum Results', 'wp-parsely' ) }
+						min="1"
+						max="99"
```

Comment:
I sent an API request without the `limit` parameter and I'm getting 9 results, but I guess it could be more had I passed another site ID. Parse.ly docs list a maximum **pagination** limit of 2000 (!!!) records, but there's no mention of the default number of results.

For the Block, I think that a sensible number by default could be 5, and we could probably lower the maximum to something like 40 or 50? (still much but maybe there's a use case?) Open to suggestions...

Reply:
I've updated the max to 25.